e107_plugins/faqs/faqs.php

```diff
@@ -333,7 +329,7 @@ function view_all_query($srch='')
 			$insert = " AND (f.faq_question LIKE '%".$srch."%' OR f.faq_answer LIKE '%".$srch."%' OR FIND_IN_SET ('".$srch."', f.faq_tags) ) ";
 
 
-			$message = "<span class='label label-lg label-info'>".$srch." <a class='e-tip' title='Remove' href='".$removeUrl."'>×</a></span>";
```

Comment:
LAN_REMOVE may be added as a common term I think, or we can replace it with LAN_DELETE? Depending on the context.